### DISCUSSION TITLE
Ability to change currency?

### DISCUSSION THREAD

**Discussion by NootNootPingu** [LucaArgentieri/Shoe-Discord-Bot#14] [Ideas]
it would be nice to have the ability to change the currency :)

**Comment by LucaArgentieri** [comment]
Oh yes, this is a good point.
At the moment the only currency is the US dollar.
We have to ask [@druv5319](https://github.com/druv5319) to update the [Sneaks-API](https://github.com/druv5319/Sneaks-API) and create an option to be able to choose which currency to use.
Once that's done, we could implement the feature

**Reply by NootNootPingu** [reply]
Awesome, Id love to see it
